site/sidebar.js

```diff
@@ -283,5 +283,13 @@ module.exports = {
                 "emergency/emergencyCallingApi",
             ],
         },
-    ],
+    {
+      type: 'category',
+      label: 'Identity Management',
+      items: [
+          'identity/about',
+          'identity/identityProvidersOverview',
+      ],
+    }
+  ],
```

Comment:
The idea here is to have a generic Identity Management section in which Identity Provider is one of the available things. It might have other things in the future...

![image](https://user-images.githubusercontent.com/77112549/152860850-5c5510b3-1683-464d-b8e3-01412bde31ed.png)